packages/website/src/components/Schedule/ScheduleItem.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import { Center } from '@coderscamp/ui/components/Center';
+import { Typography } from '@coderscamp/ui/components/Typography';
+
+interface ScheduleItemProps {
+  index: number;
+  date: string;
+  description: string;
+}
+
+export const ScheduleItem = ({ index, date, description }: ScheduleItemProps) => {
+  return (
+    <Center
+      flexDirection="column"
+      justifyContent="start"
+      textAlign="center"
+      w="min(400px, 95%)"
```

Comment:
💭 95% can cause some troubles as we would need to check all sizes of the screen to determine if it works. I think you can replace this prop with `maxW="400px"` and add some constant padding using the `px` prop of the parent `SimpleGrid` component